llvm-core/clang-linker-config/clang-linker-config-22.0.0_pre20250820.ebuild

```
@@ -0,0 +1,27 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Default linker configuration for clang"
+HOMEPAGE="https://clang.llvm.org/"
+S=${WORKDIR}
+
+LICENSE="metapackage"
+SLOT="${PV%%.*}"
+IUSE="default-lld"
+
+RDEPEND="
+	default-lld? ( ~llvm-core/lld-${PV} )
+	!default-lld? ( sys-devel/binutils )
+"
+
+src_install() {
+	insinto "/etc/clang/${SLOT}"
+	newins - gentoo-linker.cfg <<-EOF
+		# This file is initially generated by llvm-core/clang-linker.
```

```suggestion
		# This file is initially generated by ${CATEGORY}/${PN}
```

(it's outdated too, should make it easier to keep in sync)

---

Done